hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/WritableAccountStore.java

```diff
@@ -83,6 +83,7 @@ protected WritableKVState<ProtoBytes, AccountID> aliases() {
      */
     public void put(@NonNull final Account account) {
         Objects.requireNonNull(account);
+        requireNotDefault(account.accountIdOrThrow());
```

Comment:
Did this fix a bug, or just a sensible defensive check?

Reply:
There was a case where we set `NftID.DEFAULT` set in migration code previously. Just a sensible check. Will do in all stores in later PR  😃

Reply:
Ah got it. ☺️